packages/ihe-gateway/server/Channels/CareQuality Document Exchange - Outbound/XCA - DQ App Interface.xml

```diff
@@ -78,11 +102,35 @@ Last updated: Nov 30 2023</description>
             </entry>
           </resourceIds>
           <queueBufferSize>1000</queueBufferSize>
-          <reattachAttachments>false</reattachAttachments>
+          <reattachAttachments>true</reattachAttachments>
         </destinationConnectorProperties>
-        <channelId>none</channelId>
-        <channelTemplate>${message.encodedData}</channelTemplate>
-        <mapVariables/>
+        <host>http://host.docker.internal:8080/internal/carequality/document-query/response</host>
```

Comment:
You should be able to replicate the approach for XCPD and send a `URL` to this config property